seed/lib/merging/merging.py

```diff
@@ -83,7 +92,90 @@ def get_state_attrs(state_list):
         return get_taxlotstate_attrs(state_list)
 
 
-def merge_extra_data(b1, b2, default=None):
+def _merge_relationships(merged_state, state1, state2):
```

Comment:
Is it obvious this method is for PropertyState objects?  It's not in the function name or the docstring.  Obviously we know what it does, but it would be really vague for others.  It feels like this should belong on the PropertyState class instead of just hanging out here in a separate lib file, but I'm sure you have your reasons for putting it here.

Reply:
moving to PropertyState class method.